api/src/main/java/io/opentelemetry/resources/Resource.java

```diff
@@ -42,11 +43,16 @@
           + " characters.";
   private static final String ERROR_MESSAGE_INVALID_VALUE =
       " should be a ASCII string with a length not exceed " + MAX_LENGTH + " characters.";
-  private static final Resource EMPTY =
-      new AutoValue_Resource(Collections.<String, String>emptyMap());
+  private static final Resource EMPTY = create(new HashMap<String, String>());
```

Comment:
Is usage of a `HashMap` instead of `Collections.emptyMap()` related to the deadlock?

Reply:
It's a typo with copy-paste, fixed.